web/gds-user-ui/package.json

```diff
@@ -72,6 +72,8 @@
     "cy:install": "cypress install",
     "cy:run": "cypress run",
     "cy:open": "cypress open",
+    "cy:report": "cypress run --reporter mochawesome --reporter-options reportDir=cypress/reports,overwrite=false,html=false,json=true",
```

Comment:
If `cy:report` is run and then `cy:mocha`, we can get reports that look like this:
![Mochawesome Report](https://github.com/trisacrypto/directory/assets/94616884/cb598ede-dbd1-4447-ae5c-0f19d83492ba)